src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/SchedulerTab.php

```diff
@@ -0,0 +1,284 @@
+            __('Tasks in queue', I18N::DOMAIN),
+            TaskModel::count(['status = :status'], ['status' => TaskHandler::STATUS_NEW]).
+            sprintf(
+                __(
+                    ' (new: %s p/h, processed: %s p/h)',
+                    I18N::DOMAIN
+                ),
+                $incomingRate,
+                $processedRate
+            )
+        );
+
+        echo $this->getFormEnd();
+    }
+
+    private function renderAdvancedConfiguration(): void
+    {
+        $url = $this->getActionUrl(self::SAVE_ACTION);
+        echo $this->getFormstart('post', $url);
+
+        echo '<br>'.$this->getFormHeader(__('Advanced configuration', I18N::DOMAIN));
+
+        $this->renderRunner();
+        $this->renderInstructions();
+        $this->renderStatistics();
+
+        echo '<br>'.$this->getFormActionGroup(
+            $this->getFormButton(
+                __('Save settings', I18N::DOMAIN),
+                'button-primary'
+            )
+        );
+
+        echo $this->getFormEnd();
+    }
+
+    public function saveAction(): void
+    {
+        $cronRunner = CronOptions::getConstant(CronOptions::RUNNER);
+        $cronLastStatus = CronOptions::getConstant(CronOptions::LAST_EXECUTION_STATUS);
+
+        $data = [
+            $cronRunner => $_POST[$cronRunner],
+            $cronLastStatus => CronRegistrar::STATUS_UNEXECUTED,
+        ];
+
+        foreach ($data as $key => $value) {
+            update_option($key, $value);
+        }
+
+        CronOptions::resetLastExecutionData();
+
+        if (CronRegistrar::RUNNER_WPCRON !== $data[$cronRunner]) {
+            wp_clear_scheduled_hook(CronRegistrar::HOOK_PROCESS_TASK);
+        }
+
+        wp_redirect(remove_query_arg('action'));
+    }
+
+    private function renderRunner(): void
+    {
+        $cronRunner = CronOptions::getConstant(CronOptions::RUNNER);
+
+        $cronRunnerValue = CronOptions::get(CronOptions::RUNNER, CronRegistrar::RUNNER_WPCRON);
+        echo $this->getFormGroup(
+            __('Cron runner', I18N::DOMAIN),
+            $this->getFormSelect(
+                $cronRunner,
+                CronRegistrar::getCronRunners(),
+                $cronRunnerValue
+            ).' <br><small>'.__(
+                'Select which cron runner will be used. Please use only if knowledgeable.',
+                I18N::DOMAIN
+            ).'</small>'
+        );
+    }
+
+    private function renderInstructions(): void
+    {
+        $runner = CronOptions::get(CronOptions::RUNNER, CronRegistrar::RUNNER_WPCRON);
+        if (CronRegistrar::RUNNER_CRONTAB_CLI === $runner) {
+            $pluginPath = ABSPATH;
+            $allowSpawnArg = WpCliCommandRunner::ALLOW_SPAWN;
+            $commandName = ScheduledProcessor::getCommandName();
+            $commandPrefix = WpCliCommandRunner::command_prefix;
+            $instructions = [
+                __('Check if `wp-cli` is installed in the website\'s server.', I18N::DOMAIN),
```

Comment:
Can you add link to https://wp-cli.org/#installing so they don't have to google?

Reply:
Added documentation link for wp-cli.